docs/pages/includes/install-linux.mdx

```diff
@@ -1,220 +1,186 @@
+<Admonition type="tip">
+
+In the example commands below, update `$SYSTEM-ARCH` with the appropriate
+value (`amd64`, `arm64`, or `arm`). All example commands using this variable
+will update after one is filled out. This variable may not be present, depending
+on your installation type.
+
+Be sure to select the apporpriate scope at the top of this page (OSS,
+Enterprise, Cloud) to see the correct example commands for your installation
+type.
+
+</Admonition>
+
+<ScopedBlock scope={["enterprise", "cloud"]}>
```

Comment:
If we go with the approach I've suggested in the comment above, we can copy this to the "Enterprise" and "Cloud" `TabItem`s